content/generators/geometric.md

```diff
@@ -403,45 +371,36 @@ The default weights for TRGG are drawn from an exponential distribution with
 rate parameter `lambda=1`.
 
 ```{code-cell} ipython3
----
-jupyter:
-  outputs_hidden: false
----
-fig, ax = plt.subplots()
-#default TRGG weight distribution network
-TRGG = nx.thresholded_random_geometric_graph(nodes,0.1,0.0001,pos=pos,weight=weight)
-nx.draw_networkx_nodes(G, pos=pos, ax=ax, **node_opts)
-draw_edges_fast(G, pos=pos, ax=ax, **edge_opts)
-```
-
-```{code-cell} ipython3
----
-jupyter:
-  outputs_hidden: false
----
-fig, ax = plt.subplots()
-#Increased threshold parameter, theta, reduces graph connectivity
-TRGG = nx.thresholded_random_geometric_graph(nodes,0.1,0.001,pos=pos,weight=weight)
-nx.draw_networkx_nodes(G, pos=pos, ax=ax, **node_opts)
-draw_edges_fast(G, pos=pos, ax=ax, **edge_opts)
+fig, axes = plt.subplots(1, 2)
+
+# Increased threshold parameter, theta, reduces graph connectivity
+thresholds = (0.0001, 0.001)
+for thresh, ax in zip(thresholds, axes):
+    TRGG = nx.thresholded_random_geometric_graph(
+        nodes, 0.1, thresh, pos=pos, weight=weight
+    )
+    nx.draw_networkx_nodes(G, pos=pos, ax=ax, **node_opts)
+    draw_edges_fast(TRGG, pos=pos, ax=ax, **edge_opts)
+    ax.set_title(f"Threshold = {thresh}, {TRGG.number_of_edges()} edges")
+fig.tight_layout()
 ```
 
 ## References
 
-[1] - Spatial Networks https://arxiv.org/pdf/1010.0302.pdf
+[^1]: Spatial Networks <https://arxiv.org/abs/1010.0302>
 
-[2] - Random Geometric Graphs https://arxiv.org/pdf/cond-mat/0203026.pdf
+[^2]: Random Geometric Graphs <https://arxiv.org/abs/cond-mat/0203026>
 
-[3] - Unit Disk Graphs http://www.sciencedirect.com/science/article/pii/0012365X9090358O
+[^3]: Unit Disk Graphs <http://www.sciencedirect.com/science/article/pii/0012365X9090358O>
 
-[4] - Waxman Graphs - http://ieeexplore.ieee.org/document/12889/
+[^4]: Waxman Graphs <http://ieeexplore.ieee.org/document/12889/>
 
-[5] - Waxman Graph Parameter Estimation - https://arxiv.org/pdf/1506.07974.pdf
+[5]: Waxman Graph Parameter Estimation <https://arxiv.org/abs/1506.07974>
 
-[6] - Threshold Graphs - http://www.combinatorics.org/ojs/index.php/eljc/article/view/v16i1r130/pdf
+[^6]: Threshold Graphs - <http://www.combinatorics.org/ojs/index.php/eljc/article/view/v16i1r130/pdf>
 
-[7] - Soft Geometric Random Graphs - https://arxiv.org/pdf/1311.3897.pdf
+[^7]: Soft Geometric Random Graphs - <https://arxiv.org/abs/1311.3897>
 
-[8] - Geometric Threshold Graphs - https://arxiv.org/pdf/cond-mat/0409378.pdf
+[^8]: Geometric Threshold Graphs - <https://arxiv.org/abs/cond-mat/0409378>
 
-[9] - Thresholded Random Geometric Graphs - http://cole-maclean.github.io/blog/files/thesis.pdf
+[^9]: Thresholded Random Geometric Graphs - <http://cole-maclean.github.io/blog/Thresholded-Random-Geometric-Graphs>
```

Comment:
Let's go full academia with DOI links 💃 and all that jazz.

[1] https://doi.org/10.1016/j.physrep.2010.11.002
[2] https://doi.org/10.1103/PhysRevE.66.016121
[3] https://doi.org/10.1016/0012-365X(90)90358-O
[4] https://doi.org/10.1109/49.12889
[6] https://doi.org/10.37236/219
[7] https://doi.org/10.1214/15-AAP1110
[8] https://doi.org/10.1103/PhysRevE.71.036108
[9] http://hdl.handle.net/2117/111425

Reply:
Great idea! Fixed in 5d23cdd